src/test/java16/org/eclipse/yasson/records/RecordTest.java

```diff
@@ -102,11 +105,26 @@ public void testRecordParameterOptionality() {
         String json = Jsonbs.defaultJsonb.toJson(car);
         assertEquals(expected, json);
 
-        String toDeserialize = "{}";
-        String expectedDefaultValues = "{\"color\":null,\"somePrimitive\":0,\"type\":\"typeDefaultValue\"}";
-        CarCreatorOptionalTest deserialized = Jsonbs.defaultJsonb.fromJson(toDeserialize, CarCreatorOptionalTest.class);
-        json = Jsonbs.nullableJsonb.toJson(deserialized);
-        assertEquals(expectedDefaultValues, json);
     }
 
+    @Test
+    public void record_with_type_deserializer() {
```

Comment:
Underscores are rather untypical. Why not aligning with the camel case of the existing tests?